test/remote/config_update_tests.js

```diff
@@ -77,6 +77,37 @@ test(
 )
 
 test(
+  'change nested limits',
+  function (t) {
+    var x = null
+    return client.getAsync('/limits')
+      .spread(function (req, res, obj) {
+        // This is derived from uidRateLimit.maxChecks
+        return obj.maxChecksPerUid
+      })
+      .then(function (mcpuid) {
+        x = mcpuid
+        return mcHelper.setLimits({ uidRateLimit: { maxChecks: mcpuid + 1 } })
+      })
+      .then(function (settings) {
+        t.equal(x + 1, settings.maxChecksPerUid, 'helper sees the change')
+        return Promise.delay(1010)
```

Comment:
It's to wait for the server's background polling to detect the new value in memcache, and update itself accordingly.  I'll add a comment :-)